src/main/java/com/nutomic/syncthingandroid/activities/DeviceActivity.java

```diff
@@ -53,6 +55,12 @@
             "com.nutomic.syncthingandroid.activities.DeviceActivity.IS_CREATE";
 
     private static final String TAG = "DeviceSettingsFragment";
+    private static final String IS_SHOWING_DISCARD_DIALOG =
+            "com.nutomic.syncthingandroid.activities.DeviceActivity.DISCARD_FOLDER_DIALOG_STATE";
+    private static final String IS_SHOWING_COMPRESSION_DIALOG =
+            "com.nutomic.syncthingandroid.activities.DeviceActivity.COMPRESSION_FOLDER_DIALOG_STATE";
+    private static final String IS_SHOWING_DELETE_DIALOG =
+            "com.nutomic.syncthingandroid.activities.DeviceActivity.DELETE_FOLDER_DIALOG_STATE";
```

Comment:
These constants probably arent necessary. Especially the package name, as they are only available in the current activity/fragment. But theyre not a problem either, so whatever you prefer.